projects/ag-grid/src/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.ts

```diff
@@ -20,24 +23,31 @@ import {
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class SkyAgGridCellEditorLookupComponent
-  implements ICellEditorAngularComp, IPopupComponent<any>
+  implements ICellEditorAngularComp, IPopupComponent<any>, OnDestroy
 {
   public skyComponentProperties?: SkyLookupProperties;
-  public isAlive = true;
+  public isAlive = false;
   public lookupForm = new FormGroup({
     currentSelection: new FormControl({
       value: [],
       disabled: false,
     }),
   });
+  public useAsyncSearch: boolean = false;
 
+  private ngDestroy = new Subject<void>();
   private params: SkyCellEditorLookupParams;
 
   constructor(
     private changeDetector: ChangeDetectorRef,
     private elementRef: ElementRef
   ) {}
 
+  public ngOnDestroy(): void {
+    this.ngDestroy.next();
```

Comment:
I see where we create this and are tearing it down here. However, I'm not seeing where it is actually used - am I just missing that?